lms/services/oauth2_token.py

```diff
@@ -55,10 +55,7 @@ def get(self):
                 .one()
             )
         except NoResultFound as err:
-            raise CanvasAPIAccessTokenError(
-                explanation="We don't have a Canvas API access token for this user",
-                response=None,
-            ) from err
+            raise NoOAuth2Token() from err
```

Comment:
It would be good if this had something here, otherwise it'll stringify to an empty string, which can be confusing.